### PR TITLE
Feature/add best text editor to forms

### PR DESCRIPTION
**Pull request by HarisShahidHameed**
Trello-link : https://trello.com/c/5HvNuYX4/81-use-text-best-editor-in-quiz-tutorial

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@ckeditor/ckeditor5-build-classic": "^12.3.1",
```
Comment (abdulhaqshah) [inline package.json:6]: you have installed both?

File: src/containers/Dashboard/Forms/CourseForm.js [hunk 2]
```diff
@@ -5,6 +5,8 @@ import "react-notifications-component/dist/theme.css";
 import { addNotification } from "../../../utilities/index";
 import API from "../../../api/index";
 import "./CourseForm.scss";
+import CKEditor from '@ckeditor/ckeditor5-react';
+import ClassicEditor from '@ckeditor/ckeditor5-build-classic';
```
Comment (abdulhaqshah) [inline src/containers/Dashboard/Forms/CourseForm.js:9]: what classic is doing here

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]

**Review by abdulhaqshah — CHANGES_REQUESTED** [review]
@haris_shahid turn the quiz form width into full page so the tutorial is added properly.Secondly media isn't working in the text editor
image

File: package.json (hunk 1 above)
Comment (HarisShahidHameed) [reply package.json:6]: We have to call them both to work.

File: src/containers/Dashboard/Forms/CourseForm.js (hunk 2 above)
Comment (HarisShahidHameed) [reply src/containers/Dashboard/Forms/CourseForm.js:9]: Because ck5 gives you to choose from it's different editors and we have chosen classic editor.

**Review by abdulhaqshah — APPROVED** [review]

**Pull request merged**